tools/cli/bin/jetpack.js

```diff
@@ -71,5 +72,10 @@ try {
 	process.exit( 1 );
 }
 
+/**
+ * Check if the user has been asked to enable analytics tracking.
+ */
+await checkAnalyticsEnabled();
```

Comment:
I note this can result in slightly silly behavior if the very first thing you run is `jetpack cli analytics on|off` 😀

We might just decide that's enough of a rare edge case that it's not worth caring about. OTOH, if we want to care, I dropped a suggestion later.